airflow/utils/state.py

```diff
@@ -114,6 +114,16 @@ def color_fg(cls, state):
     have been interrupted; in any case, it is no longer running.
     """
 
+    terminal = finished | {
```

Comment:
Hmmm, I'm not sure I actually need a new grouping here. Given SKIPPED was in the previous list, perhaps we just need to update the doc string on the previous one and add it there

Reply:
Yeah, looking at where it's used (all 8 places!) it's better to just add to this list.